src/itest/java/uk/gov/companieshouse/pscdataapi/steps/PscDataSteps.java

```diff
@@ -683,83 +602,65 @@ public void aPSCExistsFor(String companyNumber) throws JsonProcessingException {
         assertThat(companyPscRepository.findById(NOTIFICATION_ID)).isNotEmpty();
     }
 
-    @When("a Get request is sent for {string} and {string}")
+    @When("a Get request is sent for {string} and {string} for Individual")
     public void aGetRequestIsSentForAnd(String companyNumber, String notification_id) {
         HttpHeaders headers = new HttpHeaders();
         headers.setContentType(MediaType.APPLICATION_JSON);
         headers.setAccept(Collections.singletonList(MediaType.APPLICATION_JSON));
-
-        this.contextId = "5234234234";
         CucumberContext.CONTEXT.set("contextId", this.contextId);
         headers.set("x-request-id", this.contextId);
         headers.set("ERIC-Identity", "TEST-IDENTITY");
         headers.set("ERIC-Identity-Type", "key");
         headers.set("ERIC-Authorised-Key-Roles", "*");
-
-        HttpEntity<String> request = new HttpEntity<String>(null, headers);
+        HttpEntity<String> request = new HttpEntity<>(null, headers);
 
         String uri = "/company/{company_number}/persons-with-significant-control/individual/{notification_id}";
         ResponseEntity<Individual> response = restTemplate.exchange(uri,
                 HttpMethod.GET, request, Individual.class, companyNumber, notification_id);
 
         CucumberContext.CONTEXT.set("statusCode", response.getStatusCodeValue());
         CucumberContext.CONTEXT.set("getResponseBody", response.getBody());
-
     }
 
-    @And("the Get call response body should match {string} file")
-    public void theGetCallResponseBodyShouldMatchFile(String result) throws IOException, TransformerException {
+    @And("the Get call response body should match {string} file for Individual")
+    public void theGetCallResponseBodyShouldMatchFile(String result) throws IOException {
         String data = FileCopyUtils.copyToString(new InputStreamReader(new FileInputStream("src/itest/resources/json/output/" + result + ".json")));
         Individual expected = objectMapper.readValue(data, Individual.class);
-
         Individual actual = CucumberContext.CONTEXT.get("getResponseBody");
 
         assertThat(actual.getName()).isEqualTo(expected.getName());
         assertThat(actual.getCountryOfResidence()).isEqualTo(expected.getCountryOfResidence());
         assertThat(actual.getNaturesOfControl()).isEqualTo(expected.getNaturesOfControl());
     }
 
-    @After
```

Comment:
I think we want to keep this. The tests will run without it but it is good to make sure that you don't end up with containers that aren't cleaned up properly

Reply:
This was moved the beggining of the file, to line 85, right after the "@Before". I think it makes sense to have it there